xmake/toolchains/xcode/load_appletvos.lua

```diff
@@ -30,7 +30,7 @@ function main(toolchain)
 
     -- init target minimal version
     local target_minver = toolchain:config("target_minver")
-    local target_minver_flags = (simulator and "-mappletv-simulator-version-min=" or "-mappletvos-version-min=") .. target_minver
+    local target_minver_flags = (simulator and "-mappletvsimulator-version-min=" or "-mappletvos-version-min=") .. target_minver
```

Comment:
> Flag is "-mappletv-simulator-version-min" not "-mappletvsimulator-version-min"

? not "-mappletvsimulator-version-min"? But it was set.

Reply:
@corentin35000 had the error
> error: clang: error: unknown argument '-mappletv-simulator-version-min=17.0'; did you mean '-mappletvsimulator-version-min=17.0'?
and I found no reference to `-mappletv-simulator-version-min` (the one xmake sets) on the web, looks like a typo

Reply:
Yes, I did! I also had this case for the appletvos sdk, I have the impression that internally xmake uses the appletvsimulator flag

Reply:
I got this error for appletvos and appletvsimulator: error: clang: error: unknown argument '-mappletv-simulator-version-min=17.0'; did you mean '-mappletvsimulator-version-min=17.0'?

Reply:
I feel like there are two problems